examples/snowpeak/service/snowpeak.bal

```diff
@@ -91,8 +91,8 @@ service /snowpeak on new http:Listener(port) {
             { name: "Payment", relation: "payment" }
         ]
     }
-    resource function put reservations/[string id](@http:Payload rep:Reservation reservation)
-                returns @http:Payload{mediaType: "application/json"} rep:ReservationUpdated|rep:ReservationConflict|rep:SnowpeakInternalError {
+    resource function post reservations/[string id](@http:Payload rep:Reservation reservation)
```

Comment:
This resource method change is incorrect, the `POST` resource should be the previous one